src/graphql/data/util.tsx

```diff
@@ -93,6 +93,20 @@ const URL_CHAIN_PARAM_TO_BACKEND: { [key: string]: Chain } = {
   optimism: Chain.Optimism,
 }
 
```

Comment:
could be helpful to add inline developer documentation here about how this is distinct from CHAIN_ID_TO_BACKEND_NAME (ie: has mappings for all chains that FE supports, which is a superset of BE chains)